.env

```diff
@@ -0,0 +1,24 @@
+DB_HOST=host.docker.internal
+DB_VERSION=latest
+DB_PORT=5432
+DB_USERNAME=postgres
+DB_PASSWORD=postgres
+DB_DATABASE=postgres
+EVENDEMY_PORT=8080
+
+# Mail
+MAIL_ENABLED=false
+MAIL_HOST=a
+MAIL_PORT=0
+MAIL_USERNAME=a
+MAIL_PASSWORD=a
+MAIL_ADDRESS=a
+
+CALENDAR_COMPANY=Your Company
+CALENDAR_TIMEZONE=Europe/Berlin
+CALENDAR_ORGANIZER_NAME=Evendemy
+CALENDAR_ORGANIZER_MAIL=info@evendemy.com
+
+MEETING_IMAGE_PATH=./webapp/src/assets/meeting_images
```

Comment:
since we have the images now with a separate request, it is not necessary that the images will be in the assets folder